openedx_webhooks/info.py

```diff
@@ -48,6 +48,18 @@ def get_orgs(key):
     orgs = get_orgs_file()
     return set(o for o, info in orgs.items() if info.get(key, False))
 
+def get_person_time_of_pull_request(person, created_at):
```

Comment:
In isolation, this function has nothing to do with pull requests.  It takes a person and a date, and gives you the state of the person at that date.  We happen to use it with a pull request created_at date, but there's no reason to include that fact in the function itself.

We can give this a more generic name, and the created_at parameter can be given a more generic name.